code/modules/modular_computers/hardware/job_disk.dm

```diff
@@ -229,3 +229,58 @@
 	name = "\improper FACEBUCKS disk"
 	icon_state = "cart-signal" // might need a new sprite
 	spam_delay = 5
+
+// maint spawn disks (including FACEBUCKS disk, and viruses)
+// Some of these are possibly inferior than its original version
+/obj/item/computer_hardware/hard_drive/role/random_maint_spawn
+	name = "random job disk"
+
+/obj/item/computer_hardware/hard_drive/role/random_maint_spawn/Initialize(mapload)
+	..()
+	var/static/disk_list = list(
+		/obj/item/computer_hardware/hard_drive/role/maint, // putting the same ones to make it higher spawn chance
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/maint,
+		/obj/item/computer_hardware/hard_drive/role/medical,
+		/obj/item/computer_hardware/hard_drive/role/chemistry,
+		/obj/item/computer_hardware/hard_drive/role/signal/toxins,
+		/obj/item/computer_hardware/hard_drive/role/security,
+		/obj/item/computer_hardware/hard_drive/role/quartermaster,
+		/obj/item/computer_hardware/hard_drive/role/virus/clown,
+		/obj/item/computer_hardware/hard_drive/role/virus/mime,
+		/obj/item/computer_hardware/hard_drive/role/virus/syndicate
+	)
+	var/obj/item/computer_hardware/hard_drive/role/chosen_disk = pick(disk_list)
+	chosen_disk = new chosen_disk(loc)
+	switch(chosen_disk.type) // let's not make much subtype items
+		if(/obj/item/computer_hardware/hard_drive/role/medical)
+			chosen_disk.name = "\improper Outdated Med-U disk"
+			chosen_disk.disk_flags = DISK_MED // no robot access
+		if(/obj/item/computer_hardware/hard_drive/role/chemistry)
+			chosen_disk.name = "\improper Weathered ChemWhiz disk"
```

Comment:
despise this, one don't capitalize the adjective, then just take a list of "old" adjectives and append it to the start of the name

Reply:
![image](https://user-images.githubusercontent.com/87972842/204233108-b4409746-3e17-4294-8ce5-7200a0396a60.png)

Reply:
I am not sure if you wanted it as this way, but adj is randomy too